.gitpod.yml

```diff
@@ -14,11 +14,12 @@ tasks:
     init: |
       ##
       ## install Starport
-      go install /workspace/starport/...
+      go install /workspace/starport/...      
 
     command: |
       ##
       # configure env vars.
+      npm install -g npm@7.10.0
```

Comment:
Can you please add this to the `.gitpod.Dockerfile`? This way it'll be cached in the Docker layer and build proccess won't get more time consuming.